### PR TITLE
Bez/dev bot

### PR DESCRIPTION
**Pull request by bezJac**
initial implementation

### CODE REVIEW

File: telegram_bot.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+from telegram import Update
+from telegram.ext import ApplicationBuilder, CommandHandler, MessageHandler, filters, ConversationHandler, \
+    CallbackContext
+
+
+def get_bot_token():
+    return getenv("DENTAL_AID_BOT_TOKEN")
+
+
+# TODO create separate file to handle consts
```
Comment (YehudaNevo) [inline telegram_bot.py]: I think if its few consts its more comfortable in the same file

File: telegram_bot.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    await update.message.reply_text("Conversation canceled.")
+    return ConversationHandler.END
+
+
+def is_regular_text_message(update):
+    # Custom filter function to check if the message is a regular text message
+    return update.message.text and not update.message.text.startswith('/')
+
+
+def main():
```
Comment (YehudaNevo) [inline telegram_bot.py]: It's a good idea to separate the configuration, message handlers, and the main running logic to two diff files

File: telegram_bot.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    )
+
+    app.add_handler(CommandHandler('start', start_command))
+    app.add_handler(conv_handler)
+
+    app.run_polling()
+
+
+if __name__ == '__main__':
+    main()
```
Comment (YehudaNevo) [inline telegram_bot.py]: Error Handling:
Add error handling for situations where, for example, the user inputs an incorrect phone format or skips a step.
Add more comments to describe the functionality of each method.
Implement the conversation part

File: consts.py [hunk 4]
```diff
@@ -29,3 +33,5 @@
           "4 Do you take medication?"
 
 GPT_ROLE_SETTING_MESSAGE = {"role": "system", "content": SYSTEM_ROLE}
+
+CONVERSATION_HANDLER=""
```
Comment (yoavuzan) [inline consts.py]: You do not need to save an empty constant

File: gpt_handler.py [hunk 5]
```diff
@@ -1,28 +1,31 @@
...
 
 
 def configure_gpt_api_key():
     load_dotenv()
     openai.api_key = environ.get("OPEN_AI")
 
 
-async def get_reply_from_chatgpt(context: CallbackContext) -> str:
+async def get_reply_from_chatgpt(messages_history: List[Dict[str, str]]) -> str:
     response = openai.ChatCompletion.create(
```
Comment (yoavuzan) [inline gpt_handler.py]: this is an async function, for making this function to be async you must use acreate, not create!

**Pull request merged**